CONTRIBUTING.md

```diff
@@ -318,21 +312,3 @@ As you can see, the assets are put in a directory with the post name, and that d
 It is _MUST_ to use either JPG or PNG images, and the filesize must be less than 250 KB in all cases. 
 
 Directly linking to external images is also _NOT_ allowed.
-
-## Writing Pull Requests
```

Comment:
Add description that one should use the template provided, and link to the template. Keep this section also as it is being referenced from table of contents